src/sentry/static/sentry/app/components/tooltip.tsx

```diff
@@ -207,7 +228,7 @@ const TooltipContent = styled('div')`
 
   margin: 6px;
   text-align: center;
-  ${props => props.popperStyle};
+  ${p => p.popperStyle as any};
```

Comment:
Could this be a string or is it a css object that has a toString() method on it?

Reply:
It's a `React.CSSProperties` which I would have expected to be allowed here, but unfortunately it complains it is incompatible.

if I wrapped it in `css` or something I could probably get it happy. But I didn't think it was that big of an issue so I just `as any`'d to make things happy